support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummary.tsx

```diff
@@ -89,7 +89,7 @@ function NonGiftSummary(props: NonGiftSummaryTypes) {
 				<div css={styles.listMain}>{props.promotion}</div>
 				<span css={styles.subText}>{props.priceDescription}</span>
 			</li>
-			{props.isPatron && (
```

Comment:
This was a bug, we were only rendering the "14 day free trial" bullet point in the end summary if `isPatron` was `true`,  however this logic was incorrect, we should only show the "14 day free trial" bullet point in the order summary if `isPatron` is `false`.